index.html

```diff
@@ -275,10 +275,16 @@ <h3>
                 <li>Test Suites</li>
               </ul>
             </li>
+            <li>Other VC Data Integrity proof types:
+              <ul>
+                <li>VC Data Integrity - <a href="https://or13.github.io/lds-pgp2021"></a>PGP</li>
+              </ul>
+            </li>
             <li>Test suites for verifiable credential proof types:
             <ul>
               <li>VC Data Integrity - Ed25519</li>
               <li>VC Data Integrity - BBS+</li>
+              <li>VC Data Integrity - Json Web Signatures</li>
```

Comment:
We should collapse this item to "Test Suites for WG-defined crypographic container formats and suites."